functorch/functorch/_src/utilities.py

```diff
@@ -0,0 +1,74 @@
+import torch
```

Comment:
Can we refactor partitioners to use these utilities as well?

Reply:
done